src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs

```diff
@@ -240,6 +240,64 @@ public void When_StaticResource()
 			app.Resources.Remove("StaticHeight");
 		}
 
+		[TestMethod]
+		public void When_ThemeResource()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			if (app.Resources.ThemeDictionaries.TryGetValue("Light", out var themeDictionary)
+				&& themeDictionary is ResourceDictionary dictionary)
+			{
+				dictionary["StaticRow"] = 42;
+				dictionary["StaticWidth"] = 42.0;
+				dictionary["StaticHeight"] = 44.0;
+			}
```

Comment:
Maybe you should have a _else_ here raising exceptions ;-)

Reply:
It'll fail in the rest of the test in that case, that's fine for now.